includes/configuration_managers/class-everlit-configuration-manager.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Everlit plugin configuration manager.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+require_once NEWSPACK_ABSPATH . '/includes/configuration_managers/class-configuration-manager.php';
+
+/**
+ * Provide an interface for configuring and querying the configuration of Everlit.
+ */
+class Everlit_Configuration_Manager extends Configuration_Manager {
+
+	/**
+	 * The slug of the plugin.
+	 *
+	 * @var string
+	 */
+	public $slug = 'everlit';
+
+	/**
+	 * Configure Everlit for Newspack use.
+	 *
+	 * @return bool Enforce configurability.
+	 */
+	public function configure() {
```

Comment:
Is this function necessary? I'm not seeing it used anywhere. Would there be any case we wouldn't want Everlit to be configurable? e.g. for non-hosted (OSS) Newspack sites?

Reply:
`configure` method is called within `Newspack\Configuration_Managers` class. 
https://github.com/Automattic/newspack-plugin/blob/7ec149468ba0dfe000b9ce6d8ccfa22d66fd84bc/includes/configuration_managers/class-configuration-managers.php#L94-L106

Its required in every inheriting class due to abstract keyword:
https://github.com/Automattic/newspack-plugin/blob/7ec149468ba0dfe000b9ce6d8ccfa22d66fd84bc/includes/configuration_managers/class-configuration-manager.php#L104-L109

Reply:
Got it, thanks for clarifying!